docs/examples/revocation/2. Issue Credential.md

```diff
@@ -0,0 +1,96 @@
+```"issuer_cred_rev"```.
+The ```"issuer_cred_rev"``` event has information on the issued credential and
+how it is connected to the revocation registries and some metadata.
+
+This event will fire under two circumstances:
+
+- Once a credential (with revocation support) is issued.
+- And when a credential is revoked.
+
+The state of the event will correspond with these events,
+i.e. first it will be **"issued"** and after revocation it will be **"revoked"**.
+
+Lets take a look at example event.
+
+```json
+{
+    "wallet_id": "5df42bab-6719-4c8a-a615-8086435d4de4",
+    "topic": "issuer_cred_rev",
+    "origin": "tenant faber",
+    "group_id": "GroupA",
+    "payload": {
+      "created_at": "2024-04-30T08:51:18.177543Z",
+      "cred_def_id": "QrMaE11MnC6zjKNY1pxbq8:3:CL:8:Epic",
+      "cred_ex_id": "af4bad3f-3fcc-47ab-85e6-24224dcb2779",
+      "cred_ex_version": "2",
+      "cred_rev_id": "2",
+      "record_id": "57bd9c72-fa29-4f65-bd89-4e241471073a",
+      "rev_reg_id": "QrMaE11MnC6zjKNY1pxbq8:4:QrMaE11MnC6zjKNY1pxbq8:3:CL:8:Epic:CL_ACCUM:53462552-d716-4b0b-8b5c-914a3574d2c4",
+      "state": "issued",
+      "updated_at": "2024-04-30T08:51:18.177543Z"
+    }
+}
+```
+
+Taking  a look at the payload there is a few fields we are interested in:
+
+- ```"cred_def_id"``` or **credential definition id**.
+The same id the issuer gets when creating a credential definition
+(also used when issuing credentials).
+- ```"cred_ex_id"``` or **credential exchange id**.
+This is the same exchange id that can be found in the credential exchange record,
+the protocol version is just striped here.
+- ```"cred_rev_id"``` or **credential revocation id**.
+This is the id that ties the credential to the revocation registry.
+- ```"rev_reg_id"``` or **revocation registry id**.
+This is the id of the revocation registry the credential was issued against.
+
+The ```"issuer_cred_rev"``` event is not the only place this data is available.
+Under the issuer endpoints there is an endpoint:
+```GET /v1/issuer/credentials/revocation/record```
+This endpoint will only return the payload object of the
+```"issuer_cred_rev"``` event:
+
+```json
+{
+  "created_at": "2024-04-30T08:51:18.177543Z",
+  "cred_def_id": "QrMaE11MnC6zjKNY1pxbq8:3:CL:8:Epic",
+  "cred_ex_id": "af4bad3f-3fcc-47ab-85e6-24224dcb2779",
+  "cred_ex_version": "2",
+  "cred_rev_id": "2",
+  "record_id": "57bd9c72-fa29-4f65-bd89-4e241471073a",
+  "rev_reg_id": "QrMaE11MnC6zjKNY1pxbq8:4:QrMaE11MnC6zjKNY1pxbq8:3:CL:8:Epic:CL_ACCUM:53462552-d716-4b0b-8b5c-914a3574d2c4",
+  "state": "issued",
+  "updated_at": "2024-04-30T08:51:18.177543Z"
+}
+```
+
+This endpoint has three query parameters:
+
+- ```"credential_exchange_id"```
+- ```"credential_revocation_id"```
+- ```"revocation_registry_id"```
+
+If ```"credential_exchange_id"``` is not provided **BOTH**
+the ```"credential_revocation_id"``` and ```"revocation_registry_id"```
+**MUST** be provided.
```

Comment:
Shouldn't need all caps for emphasis, we should rephase to be to the point and matter of fact. Use Note: or NB: with bold to highlight important details. In markdown I think > also gives an indented / boxed paragraph